lib/ui/hooks.dart

```diff
@@ -208,7 +214,10 @@ PointerDataPacket _unpackPointerDataPacket(ByteData packet) {
       radiusMin: packet.getFloat64(kStride * offset++, _kFakeHostEndian),
       radiusMax: packet.getFloat64(kStride * offset++, _kFakeHostEndian),
       orientation: packet.getFloat64(kStride * offset++, _kFakeHostEndian),
-      tilt: packet.getFloat64(kStride * offset++, _kFakeHostEndian)
+      tilt: packet.getFloat64(kStride * offset++, _kFakeHostEndian),
+      scrollDeltaX: packet.getFloat64(kStride * offset++, _kFakeHostEndian),
+      scrollDeltaY: packet.getFloat64(kStride * offset++, _kFakeHostEndian),
+      sentinal: packet.getFloat64(kStride * offset++, _kFakeHostEndian),
```

Comment:
The error happens when reading from the packet right? do you need to explicitly pass it or can you just ignore the last float?